include/rocksdb/c.h

```diff
@@ -320,6 +320,17 @@ extern ROCKSDB_LIBRARY_API void rocksdb_multi_get_cf(
     const size_t* keys_list_sizes, char** values_list,
     size_t* values_list_sizes, char** errs);
 
+extern ROCKSDB_LIBRARY_API unsigned int rocksdb_key_may_exist(
```

Comment:
And add a comment about exactly when caller is expected to free `*value`. Otherwise, should be a self-explanatory wrapper.